dpbench/infrastructure/benchmark.py

```diff
@@ -95,7 +113,14 @@ def _exec(
     with timer.timer() as t:
         args = get_args()
 
+    input_size = 0
+    for arg in array_args:
+        # Original size:
+        # bench.bdata[preset][arg]
```

Comment:
Is this an actual comment or commented out dead code? If dead code then please remove. Otherwise make it more clear if it is a comment.